.eslintrc.json

```diff
@@ -9,10 +9,19 @@
     "es6": true
   },
   "rules": {
-    "indent": [2, 4],
     "strict": 0,
     "import/no-extraneous-dependencies": [2, {
       "devDependencies": ["**/*.spec.js"]
-    }]
+    }],
+    "prettier/prettier": [
```

Comment:
I'd prefer to use shared settings from my config and rather change the `.editorconfig` to use 2 spaces instead.